src/test/rgw/CMakeLists.txt

```diff
@@ -151,3 +151,16 @@ if(WITH_RADOSGW_AMQP_ENDPOINT)
   add_ceph_unittest(unittest_rgw_amqp)
   target_link_libraries(unittest_rgw_amqp ${rgw_libs})
 endif()
+
+# unittest_rgw_xml
+add_executable(unittest_rgw_xml test_rgw_xml.cc)
+add_ceph_unittest(unittest_rgw_xml)
+
+target_link_libraries(unittest_rgw_xml rgw_a ${EXPAT_LIBRARIES})
+
+# unittest_rgw_xml
+add_executable(unittest_rgw_xml test_rgw_xml.cc)
```

Comment:
this `unittest_rgw_xml` is showing up twice in a row. it looks like your branch has two of the same commit somehow?

Reply:
not sure how it happened (learning how to use git is an endless journey...), but hopefully fixed now.